alerts/ldap_password_spray.conf

```diff
@@ -1,3 +1,4 @@
 [options]
 threshold_count = 1
-search_depth_min = 60
\ No newline at end of file
+search_depth_min = 60
+host_exclusions = foo.example.com,bar.example.com
```

Comment:
This host_exclusion list will need to be swapped for LDAP stage hosts, which we can get from @jdow